onnx/helper.py

```diff
@@ -1280,7 +1280,7 @@ def str_list(str_elem: Callable[[_T], str], xs: Sequence[_T]) -> str:
 def printable_dim(dim: TensorShapeProto.Dimension) -> str:
     which = dim.WhichOneof("value")
     if which is None:
-        raise TypeError(f"which cannot be {None}.")
+        return "."
```

Comment:
We can also consider to use "?" for unknown dimension? which is same as what Netron does.

Reply:
No strong opinion. I can change. If netron use ?, let's use the same.